crates/ty_python_semantic/src/semantic_index/use_def.rs

```diff
@@ -348,7 +348,7 @@ pub(crate) struct UseDefMap<'db> {
     /// In this case, the function may implicitly return `None`.
     ///
     /// This is used by `UseDefMap::can_implicit_return`.
-    scope_start_visibility: ScopedVisibilityConstraintId,
+    reachability: ScopedReachabilityConstraintId,
```

Comment:
Will this now answer "if end of scope is reachable" instead of "Whether or not the start of the scope is visible." like the existing doc-string says?